scripts/appimage/build_appimage.sh

```diff
@@ -61,6 +61,12 @@ cp -v "/usr/share/icons/breeze/breeze-icons.rcc" "appdir/usr/share/icons/breeze/
 # TODO: further down also add:
 # -e "./appdir/usr/plugins/kgraphviewerpart.so" \
 
+# add extra platform plugins
+export EXTRA_PLATFORM_PLUGINS="libqoffscreen.so;libqwayland-generic.so"
```

Comment:
who uses this env var? linuxdeploy's qt plugin?

Reply:
yes